app/components/ui/PageNotFound.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Button } from "./inline/Button/Button";
+import styles from "./PageNotFound.module.scss";
+
+const PageNotFound = () => {
+  return (
+    <div>
+      <h1>Page not found</h1>
+      <p className={styles.description}>
+        We’re sorry, but the page you’re looking for can’t be found. The URL may
+        be misspelled or the page you’re looking for is no longer available.
+      </p>
+      <Button href="/">Go to the homepage →</Button>
```

Comment:
**thought:** Not your fault, but bummer this isn't a react router dom `<Link to...>` so we wouldn't have to do a whole page reload.

Reply:
It is my fault! I built the built the Button. Also realized there's an arrowVariant prop I can use instead of this